netbox/netbox/views/generic/bulk_views.py

```diff
@@ -161,6 +161,12 @@ def get(self, request):
         # Render the objects table
         table = self.get_table(self.queryset, request, has_bulk_actions)
 
+        # Check for filterset_form on this view, if a form exists, apply to context and table, otherwise set to None
+        filterset_form = None
+        if hasattr(self, 'filterset_form') and self.filterset_form:
+            filterset_form = self.filterset_form(request.GET, label_suffix='')
+            table.filterset_form = filterset_form
```

Comment:
`filterset_form` is already initialized to `None` on the view instance. (Also, I don't believe `label_suffix` is used anymore.)
```suggestion
        if self.filterset_form:
            filterset_form = self.filterset_form(request.GET)
            table.filterset_form = filterset_form
```